### DISCUSSION TITLE
Styling error under "Clone" -dropdown button

### DISCUSSION THREAD

**Discussion by KariNarhi** [community/community#22124] [New to GitHub]
<p>The new GitHub style looks nice, but it has a little styling issue.</p>
<p>Here’s a screenshot:<br>
<img src="https://user-images.githubusercontent.com/46904974/181068956-53adb05f-b06d-4f12-bc13-a18e2076ad2c.png" alt="GitHub_styling_error" data-base62-sha1="c6Ga65Cm4UKiReRQWPRb9JoBGwF" width="364" height="55"></p>
<p>The “Open in Visual Studio” and “Download ZIP” .-buttons are mashed together without any padding, while “Open in Desktop” has padding.</p>

**Comment by yamiacat** [comment, marked as answer]
<p>Hi <a href="https://github.community/u/karinarhi">karinarhi</a>! :wave:</p>
<p>I think you may have been seeing this for a brief period when you were using the new UI, but the old style repo page.</p>
<p>The new style repo page is now rolled out to 100% of users, so this should no longer be a problem, right?</p>

**Comment by KariNarhi** [comment]
<p>Yeah, it’s fixed now. Should’ve realised it takes a little time to port new UI.</p>
